### DISCUSSION TITLE
The agent path used to collect offline data

### DISCUSSION THREAD

**Discussion by Yang6852** [PKU-Alignment/omnisafe#288] [Q&A] [answered]
Hi,
I'm collecting data as written in the tutorial(https://github.com/PKU-Alignment/omnisafe/blob/main/benchmarks/offline/README.md) to make an offline dataset.I install the omnisafe and train the agents.When i use the sample code to collect offline data,an error is reported.
![image](https://github.com/PKU-Alignment/omnisafe/assets/66681868/7607c8f5-8fe4-4691-94da-0d5ac91d0a98)
My file directory is as follows
![image](https://github.com/PKU-Alignment/omnisafe/assets/66681868/25032b33-921c-4435-9e85-d36588cff58b)
Thanks

**Comment by Gaiejj** [comment, marked as answer]
Sorry for the bug in the example. Add ``.pt`` after ``epoch-500`` may resolve. We will fix it in the following pull request.

**Reply by Yang6852** [reply]
Thanks for your reply.I change the code as what you said,and Error 21 is resolved.However, a new error appears.Maybe there's something wrong with the Module.load_state_dict( )  definition.
![image](https://github.com/PKU-Alignment/omnisafe/assets/66681868/aa5c5e7d-de00-4ced-bf69-ce7ce63fd4ef)

**Reply by Gaiejj** [reply]
I think the version of `torch` may be the key. You can use `pip list` in your conda environment to check your version. My torch version is 2.1.0.

**Reply by Yang6852** [reply]
I got the data finally.Thanks for your attention and patience.
